SupportedServices.md

```diff
@@ -196,6 +197,13 @@ Note that using the defaults here for all alarms is probably not that useful.
 - `LatencyHigh`: 0.5 (average in seconds)
 - `UnHealthyHostCountHigh`: 1 (count)
 
+### Alb
+
+- `Alb5xxErrorsHigh`: 10 (count)
+- `AlbTarget5xxErrorsHigh`: 10 (count)
+- `AlbRejectedConnectionCountHigh`: 10 (count)
+- `AlbTargetResponseTimeHigh`: 10 (seconds) compared against `p99`
```

Comment:
this seems a bit high default for p99? maybe 1 or 2 second default?

Reply:
Changed it to 2sec.